src/foundry/foundry.js/quadtreeCulling.d.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * An experimental approach to culling using the quadtree mapping rather than the screen rectangle.
+ * @internal
+ */
+declare class QuadtreeCulling {
+  constructor();
+
+  cull(screen: Rectangle): void;
+
+  _getRect(screen: Rectangle): NormalizedRectangle;
```

Comment:
should probably be marked `@internal` and made `protected`

Reply:
I thought that wasn't needed because the entire class is already `@internal`, but sure I can add that.